etl/steps/data/meadow/un_sdg/2022-05-26/un_sdg.py

```diff
@@ -86,23 +96,20 @@ def run(dest_dir: str) -> None:
     )
     ds.add(tb)
     ds.save()
+    log.info("un_sdg.end")
 
 
 def create_dataframe(original_df: pd.DataFrame) -> None:
     # Removing the square brackets from the indicator column
-    new_columns = []
-    for k in original_df.columns:
-        new_columns.append(re.sub(r"[\[\]]", "", k))
+    original_df = original_df.copy(deep=False)
```

Comment:
I'm copying it here to get rid of `SettingsWithCopyWarning` in pandas. These warnings would otherwise show up in our alerting for etl.

(It'd be also possible to silence them globally if there were too much of them)